### PR TITLE
chore(cargo): Update Dependency 

### LINKED ISSUE #1134: chore(blink): blink not always initialized correctly
cargo run -- --path=/Users/<profile>/.uplink_test1
cargo run -- --path=/Users/<profile>/.uplink_test2

make two new accounts with those names test1 and test2

In both enable developer tools

try to call from one to the other, if it works, great

if it doesn't, stop and restart just the uplinks with the red blink initialization error in devtools
usually one works and one doesn't for me the first start, then it takes 1 or 2 restarts of one for the second to work

### PR DESCRIPTION
**Pull request by dariusc93**
### What this PR does 📖

- Relocate logic in initialization

### Which issue(s) this PR fixes 🔨

- Resolves #1134

### CODE REVIEW

**Comment by WanderingHogan** [comment]
this rocks, this makes the pieces of warp that don't load retry and load after a few moments. It fixes this ticket: https://github.com/Satellite-im/Uplink/issues/1134

We will probably want to find a way to visually show if warp has loaded or not but this is great!

**Review by sdwoodbury — APPROVED** [review]

**Review by phillsatellite — APPROVED** [review]

**Comment by phillsatellite** [comment]
Good by me, do not merge until Sara also approves it when she gets back from dinner and is able to finish her testing

**Review by stavares843 — APPROVED** [review]

**Pull request merged**
